meetups/2023/2023-09-21-talks/program.md

```diff
@@ -0,0 +1,41 @@
+# Rust Aarhus meetup at Concordium
+Date: 2023-09-21
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+We are still working on the program, but it will be something like the following:
+
+- 18:05 - 18:15: Welcome (10 min)
+- 18:20 - 18:50: "Duke nukem" by Jeppe and Rasmus (30 min)
+- 18:50 - 19:00: A word from our sponsor (5-10 min)
+- 19:00 - 19:45: Light dinner sponsored by Concordium
+- 19:45 - 20:30: "Concordium talk" by Concordium (45 min)
+- 20:35 - 20:45: "An innocent story of contributing to a Rust project" by Ole Friis Østergaard (10 min)
+- 20:45 - 20:55: Book lottery by Niels Skovmand (10 min)
+- 20:55 - 21:00: Closing remarks (5 min)
+- 21:00: Let's continue the discussion at the nearby bar [INSERT BAR NAME].
+
+## [TITLE OF DUKE NUKEM TALK]
+[INFORMATION ABOUT THE SPEAKERS]
+
+[ADD DESCRIPTION OF THE TALK HERE]
+
+## [TITLE OF CONCORDIUM TALK]
+[INFORMATION ABOUT THE SPEAKERS]
+
+[ADD DESCRIPTION OF THE TALK HERE]
+
+## An innocent story of contributing to a Rust project
+[Ole Friis Østergaard][ofo]: [INFORMATION ABOUT THE SPEAKERS]
+
+[ADD DESCRIPTION OF THE TALK HERE]
+
+## Location
+This meetup will be at [INSERT ADDRESS HERE]. Please feel free to use the event chat on meetup to organize transportation if needed.
+
+## Food
+If you have special needs regarding food (allergies, vegan, etc.), please forward your request directly to [INSERT NAME OF HOST].
```

Comment:
```suggestion
If you have special needs regarding food (allergies, vegan, etc.), please forward your request directly to philipkristoffersen@gmail.com.
```

I can be responsible for this, since I am vegetarian myself anyways.